README.md

```diff
@@ -153,7 +153,7 @@ The default priority level for interrupts is 0, so for *every single interrupt*
 ```rust
 use embassy_nrf::interrupt::{self, InterruptExt};
 
-let mut irq = interrupt::take!(SPIM3);
+let mut irq = interrupt::SPIM3;
 irq.set_priority(interrupt::Priority::P3);
 let mut spim = spim::Spim::new( p.SPI3, irq, p.P0_13, p.P0_16, p.P0_15, config);
```

Comment:
```suggestion
interrupt::SPIM3.set_priority(interrupt::Priority::P3);
let mut spim = spim::Spim::new(p.SPI3, Irqs, p.P0_13, p.P0_16, p.P0_15, config);
```

Reply:
(the `irq` is no longer what you pass to `Spim::new()`, it's the `Irqs` token from `bind_interrupts!`)

Reply:
Ahhhh, I see. Apologies. I have committed the changes you suggested.